### PR TITLE
Add scenaries that need pre and post condition

### PR DESCRIPTION
**Pull request by cristianfchq**

### CODE REVIEW

File: src/test/resources/features/createContract.feature [hunk 1]
```diff
@@ -27,3 +27,158 @@ Feature: Create Contract
       | Description             | Description         |
     Then I verify Contract created with all fields
       And I verify that the date matches the creation date
+
+  @CreateContract
+  Scenario: create a contract with Test Account as pre-postcondition
```
Comment (SaulCM) [inline src/test/resources/features/createContract.feature]: Sentences should start in capital letter.

File: src/test/resources/features/createContract.feature [hunk 2]
```diff
@@ -27,3 +27,158 @@ Feature: Create Contract
...
+  @CreateContract
+  Scenario: create a contract with Account Name and price book as pre-postcondition
+    Given I navigate to the "Contract" page
+    When I create a new contract with all fields
+      | Account Name            | TestAccount         |
+      | Contract Term (months)  | 2                   |
+      | Contract Start Date     | 8/15/2021           |
+      | Price Book              | Standard            |
+      | Owner Expiration Notice | 30 Days             |
+    Then I verify Contract created with all fields
```
Comment (SaulCM) [inline src/test/resources/features/createContract.feature]: Could be more descriptive like I  verify that the created Contract contains the correct information.

File: src/test/resources/features/createContract.feature (hunk 1 above)
Comment (cristianfchq) [reply src/test/resources/features/createContract.feature]: Ok, It was solved

File: src/test/resources/features/createContract.feature (hunk 2 above)
Comment (cristianfchq) [reply src/test/resources/features/createContract.feature]: Ok, It was solved, thanks for the suggestion

File: src/test/java/org/salesforce/steps/featureSteps.java [hunk 3]
```diff
@@ -57,10 +57,10 @@ public void iCreateANewWorkTypeOnlyWithRequiredFields(String nameFeature, final
         createdFeature=featureNew.clickSaveButton();
     }
 
-    @Then("^I verify (?:.*) created with (?:.*)$")
-    public void iVerifyWorkTypeCreatedWithRequirementFields() {
+    @Then("^I verify that the created (?:.*) contains the correct information")
```
Comment (GustavoZHA) [inline src/test/java/org/salesforce/steps/featureSteps.java:60]: Do more generic the description of step

**Review by GustavoZHA — APPROVED** [review]

**Review by SaulCM — APPROVED** [review]

**Pull request merged**